lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPlugin.java

```diff
@@ -68,17 +73,28 @@
 	private RepositorySystemSession repositorySystemSession;
 	private MavenPluginManager mavenPluginManager;
 
+	private XMLExtensionsRegistry currentRegistry;
+
+	private InitializeParams params;
+
 	public MavenPlugin() {
 	}
 
 	@Override
 	public void doSave(ISaveContext context) {
-
+		if (context.getType() == SaveContextType.SETTINGS) {

```

Comment:
We could lose one of the added imports by checking `if (context.getType() == ISaveContext.SaveContextType.SETTINGS)`. Not a big deal however.